index.html

```diff
@@ -4,63 +4,146 @@
+              <li class="nav-item">
+                <a class="nav-link" href="#contact">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <div class="date">
+        <p class="display-date"></p>
+      </div>
+    </header>
 
-      <table class="table table-striped">
-        <thead>
-          <tr>
-            <th>Title</th>
-            <th>Author</th>
-            <th>ID</th>
-            <th></th>
-          </tr>
-        </thead>
-        <tbody id="book-list"></tbody>
-      </table>
+    <main>
+      <section id="list" class="section-list hide">
+        <div class="container mt-4">
+          <h1 class="display-4 text-center">
+            <span class="text-secondary">all</span>
+            Awesome <span class="text-secondary">BOOKS</span>
+          </h1>
+          <ul class="book-list"></ul>
 
-      <form class="book-form">
-        <div class="form-group">
-          <input
-            type="text"
-            id="title"
-            name="title"
-            placeholder="title"
-            required
-          />
+          <table class="table table-striped">
+            <thead>
+              <tr>
+                <th>Title</th>
+                <th>Author</th>
+                <th>ID</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody id="book-list"></tbody>
+          </table>
         </div>
-        <div class="form-group">
-          <input
-            type="text"
-            id="author"
-            name="author"
-            placeholder="author"
-            required
-          />
+      </section>
+
+      <section id="addnew" class="section-add-book">
+        <div class="container mt-4">
+          <h2>Add a new <span class="text-secondary">BOOK</span></h2>
+          <form class="book-form">
+            <div class="form-group">
+              <input
+                type="text"
+                id="title"
+                name="title"
+                placeholder="title"
+                required
+              />
+            </div>
+            <div class="form-group">
+              <input
+                type="text"
+                id="author"
+                name="author"
+                placeholder="author"
+                required
+              />
+            </div>
+            <button class="btn btn-primary btn-block" type="submit">
+              Add
+            </button>
+          </form>
+        </div>
+      </section>
+
+      <section id="contact" class="contact-section hide">
+        <div class="container mt-4">
+          <h2>Contact Information</h2>
+          <p>
+            Do have any question or you just want to say "hello" <br />
+            You can Reach out to us!
+          </p>
+          <ul>
+            <li>Our E-mail: books@awesome.com</li>
+            <li>Our Phone Number: +964-772-218-0404</li>
+            <li>Our Adress: Sulaymaniyah-Kurdistan-Iraq</li>
```

Comment:
Kindly add a left margin to this text to make your project stick to the template provided.:pray: :pray: 
Your design
![image](https://user-images.githubusercontent.com/98363075/200089128-094945ae-a5a4-4918-bc6d-0b1e2c442c2b.png)

Template provided
![image](https://user-images.githubusercontent.com/98363075/200089186-6ed9d1fa-569a-42c0-a41c-9b62a77ceed5.png)

Reply:
Good job following the advice of the previous reviewer